examples/features/dns/README.md

```diff
@@ -17,26 +17,16 @@ kubectl wait --for=condition=ready --timeout=1m pod ${WH} -n nsm-system
 
 1. Create test namespace:
 ```bash
-NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/35ffecc5f0bdb88f80bbce0a4423cf2ab9369540/examples/features/namespace.yaml)[0])
+NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/222525f00653c70b13cda457e87cc6cbfbaae332/examples/features/namespace.yaml)[0])
```

Comment:
Why is this updated? Could you please use main version of these files?

Reply:
ok, done